query-engine/connector-test-kit/src/test/scala/queries/relations/Repro.scala

```diff
@@ -0,0 +1,53 @@
+package queries.relations
+
+import org.scalatest.{FlatSpec, Matchers}
+import util.{ApiSpecBase, ProjectDsl}
+
+class Repro extends FlatSpec with Matchers with ApiSpecBase {
```

Comment:
Rename to something more understandable, then it's alright from my side. (e.g. DuplicateColumnRegressionSpec or whatever).